### PR TITLE
Set python packages version for dependabot.

### LINKED ISSUE #1021: Use dependabot
Implement [dependabot](https://github.com/dependabot) and:
- move requirements into a `requirements` directory
- follow similar implementation as https://github.com/pyansys/pymapdl/blob/main/.github/dependabot.yml

This will help solve issues related to upstream dependencies breaking doc generation and unit testing.

### PR DESCRIPTION
**Pull request by MaxJPRey**
Add dependabot.
Set python packages version for dependabot.
Fix #1021 .

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
 # https://github.com/mhammond/pywin32
 # License: PSF
 
-pythonnet >= 2.4.0; python_version > '2.7'
+pythonnet==2.5.2; python_version > '2.7'
 # https://github.com/pythonnet/pythonnet
 # License: MIT
 
-rpyc >= 5.0.1; python_version > '2.7'
+rpyc==5.1.0; python_version > '2.7'
```
Comment (maxcapodi78) [inline requirements.txt:12]: I would need to check if this version is compatible with the one embedded in pyaedt for ironpython usage. Should be ok but it good to have a try on linux machine next week before closing it or simply stay with 5.0.1 for now.

**Review by akaszynski — APPROVED** [review]
LGTM

**Review by maxcapodi78 — COMMENTED** [review]
LGTM apart the rpyc comment

File: requirements.txt (hunk 1 above)
Comment (akaszynski) [reply requirements.txt:12]:
```suggestion
rpyc==5.0.1; python_version > '2.7'
```

**Comment by akaszynski** [comment]
I downgraded pyrc, we can wait for dependabot to suggest.

**Pull request merged**